datastore/metadata.go

```diff
@@ -0,0 +1,42 @@
+package datastore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ONSdigital/dp-mongodb/v3/mongodb"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type PopulationTypeMetadata struct {
+	// This ID refers to the Population Type
+	ID               string `json:"id" bson:"id"`
+	DefaultDatasetID string `json:"default-dataset-id" bson:"default-dataset-id"`
+}
+
+// GetMetadataRecord gets the metadata stored against a particular population type
+// currently only just the default dataset
+func (c *MongoClient) GetMetadataRecord(ctx context.Context, populationType string) (*PopulationTypeMetadata, error) {
```

Comment:
I don't think we need to specify 'record', rather keep in line with how we name other functions i.e `GetResource`.. if 'metadata' is too vague maybe call it populationMetadata, ptypeMetadata etc

Reply:
`defaultDatasetMetadata`?